tests/testthat/test-carehomes-multistrain.R

```diff
@@ -685,3 +686,41 @@ test_that("If prob_strain is NA then Rt is NA ", {
   expect_true(all(is.na(rt_all$eff_Rt_general[-1L, ])))
 
 })
+
+
+test_that("calculate Rt with both second variant and vaccination", {
+  ## Seed with 10 cases on same day as other variant
+  ##
+  ## run model with unvaccinated & vaccinated (with susceptibility halved)
+  ## waning_rate default is 0, setting to a non-zero value so that this test
+  ## passes with waning immunity
+  set.seed(1)
+  reduced_susceptibility <- 0.2 # can put anything <1 here
+  p <- carehomes_parameters(sircovid_date("2020-02-07"), "england",
+                            strain_transmission = c(1, 0.1),
+                            strain_seed_date =
+                              rep(sircovid_date("2020-02-07"), 2),
+                            strain_seed_value = 10,
+                            rel_susceptibility = c(1, reduced_susceptibility),
+                            rel_p_sympt = c(1, 1),
+                            rel_p_hosp_if_sympt = c(1, 1),
+                            waning_rate = 1 / 20)
```

Comment:
Possibly want doses specified here? Otherwise I think there's no actual vaccination going on in this simulation